pyproject.toml

```diff
@@ -14,6 +14,7 @@ keywords = ["Scientific Visualization", "shaders", "animation", "simulation",
 authors = [
     {name = "Eleftherios Garyfallidis", email = "fury@python.org"},
     {name = "Serge Koudoro", email = "fury@python.org"},
+    {name = "Maharshi Gor", email = "fury@python.org"},
```

Comment:
I would create a separate PR for that because you need to add it in more places. in the doc.